<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SilCamController</class>
 <widget class="QDialog" name="SilCamController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acquisition control</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QPushButton" name="pb_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_live_raw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Live raw</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_DriveMonitor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Drive monitor</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
